client/src/main/java/sjt/http/client/module/request/HttpRequest.java

```diff
@@ -0,0 +1,134 @@
+package sjt.http.client.module.request;
+
+import sjt.http.client.module.HttpMethod;
+
+import java.io.BufferedOutputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Map;
+
+public class HttpRequest {
+
+    private static final String HEADER_DELIMITER = ": ";
+
+    private RequestLine requestLine;
+    private Map<String, String> requestHeader;
+    private String requestBody;
+
+    public HttpRequest (HttpMethod httpMethod, String uri, String version) {
+        requestLine = new RequestLine(httpMethod, uri, version);
+    }
+
+    /**
+     * BufferedReader로 부터 request 읽어오기
+     *
+     * @param reader
+     * @throws IOException
+     */
+    public void readHttpRequest(BufferedReader reader) throws IOException {
+        requestLine = new RequestLine(reader.readLine());
+        readHeader(reader);
+        readBody(reader);
+    }
+
+    /**
+     * BufferedReader로부터 Header 정보를 읽어와 Map에 넣어준다.
+     *
+     * @param reader
+     * @throws IOException
+     */
+
+    private void readHeader(BufferedReader reader) throws IOException {
+        String requestHeaderLine;
+        while (reader.ready() && (requestHeaderLine = reader.readLine()) != null) {
+            if("".equals(requestHeaderLine)){
```

Comment:
Reformat 은 사소하지만 공동작업하는 영역에서 만약 다른 개발자가 정렬을 하게 된다면 상관없는 부분까지 diff 의 범위가 넓어질 수도 있다고 생각합니다.
```suggestion
            if ("".equals(requestHeaderLine)){
```